src/SFA.DAS.Reservations.Web/Views/Error/AccessDenied.cshtml

```diff
@@ -8,7 +8,6 @@
 
             <h1 class="govuk-heading-xl">Access denied</h1>
 
-
             @if (Configuration["AuthType"].Equals("provider", StringComparison.CurrentCultureIgnoreCase))
             {
                 <p class="govuk-body">You don’t have permission to access the Apprenticeship Service.</p>
```

Comment:
I think you could simply swap out the content within this ```if()``` with the new content. This would only show for Access denied for a provider in this case.

The other changes could be reverted as they are employer specific and the content outside the ```if()``` would remain unchanged